lib/chef-dk/command/generator_commands.rb

```diff
@@ -37,19 +37,22 @@ module SharedGeneratorOptions
         :short => "-I LICENSE",
         :long => "--license LICENSE",
         :description => "all_rights, apache2, mit, gplv2, gplv3 - defaults to all_rights",
-        :default => "all_rights"
+        :default => "all_rights",
```

Comment:
The reason I used proc was to maintain parody with the way knife does it.  What is the advantage of putting the config read in default?